pkg/k8sutil/deployments.go

```diff
@@ -0,0 +1,22 @@
+package k8sutil
+
+import (
+	"context"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func DeleteDeployment(ctx context.Context, k8sclientset kubernetes.Interface, clusterNamespace string, deployment string) {
```

Comment:
instead of new file, put this function on file https://github.com/rook/kubectl-rook-ceph/blob/master/pkg/k8sutil/k8sutil.go